src/components/Typography/__tests__/index.test.tsx

```diff
@@ -1,22 +1,26 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
 import React from 'react'
 import ReactDOM from 'react-dom'
 import Typography, { typographyVariants } from '..'
 import shouldMatchEmotionSnapshot from '../../../helpers/shouldMatchEmotionSnapshot'
 
+// TODO: Remove any but IDK what we can put here
 jest.mock(
   '../../Tooltip',
   () =>
-    ({ children, ...props }) =>
+    ({ children, ...props }: { children: (props: unknown) => unknown }) =>
```

Comment:
What about `props: { [x: string]: unknown }` ? It's a bit same but at least it precise the structure